src/core_plugins/kibana/public/home/components/tutorial/footer.js

```diff
@@ -47,6 +59,10 @@ export function Footer({ overviewDashboard }) {
 }
 
 Footer.propTypes = {
+  launchApp: PropTypes.shape({
```

Comment:
What do you think of this footer just taking label and url attributes, then doing the logic externally of what to send in and whether or not to render a footer?  I think that would better illustrate what this react component really cares about (not where the url and label to the button come from, but only what they are).

Reply:
Nice idea. I will move the launch/overviewDashboard to an external component to keep the footer nice and simple.